components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/RequestObjectValidatorUtil.java

```diff
@@ -245,4 +289,22 @@ public static boolean isSignatureVerified(SignedJWT signedJWT, Certificate x509C
             return false;
         }
     }
+
+    public static JWSVerifier getVerifier(PublicKey publicKey) {
+
+        return new RSASSAVerifier((RSAPublicKey) publicKey);
+
+    }
+
+    public static boolean isFapiConformant(String clientId) throws RequestObjectException {
+
+        try {
+            return OAuth2Util.isFapiConformantApp(clientId);
+        } catch (IdentityOAuth2Exception e) {
+            log.debug("Error while retrieving service provider. Unable to verify whether the service provider is " +
```

Comment:
Since we are throwing an with the same info exception why do we need the debug log here?

Reply:
Fixed in f409e32a5e8cad00a5dfeaaac027011720f2c534